utilities/titandb/titan_db_test.cc

```diff
@@ -237,19 +298,56 @@ TEST_F(TitanDBTest, DbIter) {
 }
 
 TEST_F(TitanDBTest, Snapshot) {
-    Open();
-    std::map<std::string, std::string> data;
-    Put(1, &data);
-    ASSERT_EQ(1, data.size());
+  Open();
+  std::map<std::string, std::string> data;
+  Put(1, &data);
+  ASSERT_EQ(1, data.size());
 
-    const Snapshot* snapshot(db_->GetSnapshot());
-    ReadOptions ropts;
-    ropts.snapshot = snapshot;
+  const Snapshot* snapshot(db_->GetSnapshot());
+  ReadOptions ropts;
+  ropts.snapshot = snapshot;
 
-    VerifyDB(data, ropts);
-    Flush();
-    VerifyDB(data, ropts);
-    db_->ReleaseSnapshot(snapshot);
+  VerifyDB(data, ropts);
+  Flush();
+  VerifyDB(data, ropts);
+  db_->ReleaseSnapshot(snapshot);
+}
+
+TEST_F(TitanDBTest, IngestExternalFiles) {
+  Open();
+  SstFileWriter sst_file_writer(EnvOptions(), options_);
+  ASSERT_EQ(sst_file_writer.FileSize(), 0);
+
+  const uint64_t kNumEntries = 100;
+  std::map<std::string, std::string> data;
+  for (uint64_t i = 1; i <= kNumEntries; i++) {
+    Put(i, &data);
+  }
+  ASSERT_EQ(kNumEntries, data.size());
+  Flush();
```

Comment:
also verify db before flush?

Reply:
other similar tests doesn't, need to update them as well?

Reply:
specifically TitanDBTest::Basic

Reply:
no, just this test. here I want to verify data is correct before and after flush.